package.json

```diff
@@ -141,6 +141,7 @@
     "typescript": "4.4.4"
   },
   "resolutions": {
+    "react-dom": "17.0.2",
```

Comment:
nitpick: I don't have a better solution to this other than maybe adding the dependency to `devDependencies`. But I'm also not sure if that will fix the warning.